### ISSUE TITLE
Inconsistent results in the same lookup across RT's

### ISSUE THREAD

**Issue by sfolsom** [LD4P/sinopia_editor#1113]
**Describe the bug**
What went wrong? What did you expect to happen?

We are experiencing different results for the same lookup across different RT. 

E.g. in the Primary Contribution Property template from CornellSinatra45:Work on https://stage.sinopia.io/templates we see:

![Screen Shot 2019-07-31 at 11 42 23 AM](https://user-images.githubusercontent.com/8257258/62234399-9ce27b00-b398-11e9-94fd-4e2cb6b08a77.png)

As compared to the LOC person [names] Property Template in Harvard:rt:test:lookups on https://stage.sinopia.io/templates, where we see:

![image (1)](https://user-images.githubusercontent.com/8257258/62234455-b4216880-b398-11e9-810b-75c74929d96a.png)

- What is the impact on your work? (can't work / work stoppage; workaround exists; minor issue)

Can't work... hard to test RTs, or begin cataloging if we don't know how lookups work. The work around is to search outside of Sinopia and add a URI, but that would be less than ideal.

- What environment are you in? (stage or production) 

stage

- What OS and browser are you using? (Mac/Windows; Safari, Firefox, etc.)

Chrome

- If you're cataloging, what Resource Template were you using (name or ID)

see above.

**Please include a screenshot**
(Mac: shift-command-4; Windows: ; then drag the image file into the issue)

see above.

**Comment by jcoyne** [comment]
I can't duplicate on stage, when I click: "There was a problem retrieving CornellSinatra45:bf2:Agent:RWO:FieldofActivity."

The console is showing errors like this:
<img width="977" alt="Screen Shot 2019-07-31 at 2 29 13 PM" src="https://user-images.githubusercontent.com/92044/62241882-a28f8f00-b39f-11e9-9507-3b95eb2d30a2.png">

**Comment by jcoyne** [comment]
It looks like Trellis is returning a 204 or 404 for some of these requests:
<img width="1640" alt="Screen Shot 2019-07-31 at 2 32 13 PM" src="https://user-images.githubusercontent.com/92044/62242095-0dd96100-b3a0-11e9-9f33-2c94e7f6cc65.png">

**Comment by jermnelson** [comment]
Hi @sfolsom, is the `CornellSinatra45:bf2:Agent:RWO:IdentifiedbyAuthority` resource template somewhere we can retrieve it? I wasn't able to find it in the Cornell directory in the [Sinopia Sample Profiles](https://github.com/LD4P/sinopia_sample_profiles) repository. Thanks.

**Comment by sfolsom** [comment]
@jermnelson @jcoyne  I've removed the Property Template that referred to `CornellSinatra45:bf2:Agent:RWO:IdentifiedbyAuthority` because I'm not sure what LOC is attempting to do with it. I've also added some of the other Resource Templates that were missing that were referred to by the Primary Contributor template. 

Long-story-short... I'm now getting the same lookup results (depicted in the second screenshot above) across both Resource Templates. I think we can close this issue until I find another example.

P.S. I'd love to hear why the app is more sensitive to missing embedded Resource Templates today than it was yesterday. (I realize I was going to eventually have to add them, but it was helpful to see a top-level resource template in Sinopia before deciding on/completing the embedded ones.)
